### ISSUE TITLE
No error but no Update on Discord either

### ISSUE THREAD

**Issue by zeroy99** [TibRib/Perforce-discord-botlogger#1]
Hi, followed instructions, no error on running the app.py - also tested the Webhook URL and everything is fine but when I run the Python script I get no update on discord.. how can I troubleshoot further? any logging? Thanks!

**Comment by TibRib** [comment]
Hello zeroy, I guess you have set the right changelist number inside last_change.ini,
Could you try changing it to a lower value and running the script with a powershell ? You should see a message saying "Sent for dispatch" if the CLs are correctly picked up

**Comment by zeroy99** [comment]
Yeah had correct change number in the INI and could see the commands from Perforce Log too.. currently using James Ives script for now as it works, formatting not as nice as yours though... I'll test in Powershell and see! Thanks

**Comment by TibRib** [comment]
Hum, I will try to repro

**Comment by TibRib** [comment]
Was your perforce repository root or under a certain path ?

**Comment by zeroy99** [comment]
I took the path from the Perforce log directly and tried a few depth on that too - Im running P4VS on Windows Server 2019 btw

**Comment by zeroy99** [comment]
I fixed the issue now by simply removing {repository} in that line:
`p4_changes = subprocess.Popen(f'p4 changes -t -m {max} -s submitted -e {self.read_num()+1} -l', stdout=subprocess.PIPE, shell=True)`

I think it may be dude to the fact that I use a StreamDepot, not a Depot?
